source/_integrations/group.markdown

```diff
@@ -42,20 +42,59 @@ Home Assistant can group multiple binary sensors, covers, fans, lights, locks, m
 
 ## Group behavior
 
-By default, when any group member entity is `on`, the group will also be `on`. A complete overview of how groups behave:
+### Binary sensor, light and switch groups
 
-- The group state is `on` if at least one group member is `on`.
-- Otherwise, the group state is `unavailable` if all group members are `unavailable`.
+In short, when any group member entity is `on`, the group will also be `on`. A complete overview of how groups behave:
+
+- The group state is `unavailable` if all group members are `unavailable`.
 - Otherwise, the group state is `unknown` if all group members are `unknown`.
+- Otherwise, the group state is `on` if at least one group member is `on`.
```

Comment:
This should be on top, as was changed before based on comments from others as well.

Right now the unavailable state is listed first; which is odd as it is an exceptional state.